### PR TITLE
[csharp] Add GenHTTP framework

### PR DESCRIPTION
**Pull request by Kaliumhexacyanoferrat**
Struggling a little bit with the documentation but this should probably work :)

### CODE REVIEW

File: csharp/genhttp/web.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <AssemblyName>web</AssemblyName>
+    <OutputType>Exe</OutputType>
+
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+    <TieredPGO>true</TieredPGO>
+
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GenHTTP.Core.Kestrel" Version="9.1.0" />
```
Comment (waghanza) [inline csharp/genhttp/web.csproj:16]:
```suggestion
    <PackageReference Include="GenHTTP.Core.Kestrel" Version="9.1.*" />
```

File: csharp/genhttp/web.csproj [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <PackageReference Include="GenHTTP.Modules.Functional" Version="9.1.0" />
```
Comment (waghanza) [inline csharp/genhttp/web.csproj:17]:
```suggestion
    <PackageReference Include="GenHTTP.Modules.Functional" Version="9.1.*" />
```

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by waghanza** [comment]
Few minor changes.

Thanks for your contribution :tada:

**Comment by Kaliumhexacyanoferrat** [comment]
Thanks - switched to wildcard references in the latest commit :)

**Review by waghanza — APPROVED** [review]

**Pull request merged**